howto/setup-secret-store/json-secret-store.md

```diff
@@ -0,0 +1,104 @@
+
+1. Create a components directory in your application root
+
+All Dapr components are stored in a directory called 'components' below at application root. Create this directory.
+
+```bash
+mkdir components
+```
+
+2. Create a file called jsonsecretstore.yaml in the components directory
+
+Now create an Dapr jsonsecretstore component. Create a file called jsonsecretstore.yaml in the components directory with the content below
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: jsonsecretstore
+  namespace: default
+spec:
+  type: secretstores.json.jsonsecretstore
+  metadata:
+  - name: secretsFile
+    value: [path to the secrets.json you created in the previous section]
+  - name: nestedSeparator
+    value: ":"
+```
+
+3. Create redis.yaml in the components directory with the content below
+
+Create a statestore component file. This Redis component yaml shows how to use the `redisPassword` secret stored in a JSON secret store called jsonsecretstore as a Redis connection password.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+  namespace: default
+spec:
+  type: state.redis
+  metadata:
+  - name: redisHost
+    value: "[redis]:6379"
+  - name: redisPassword
+    secretKeyRef:
+      name: redisPassword
+      key: redisPassword
+auth:
+    secretStore: jsonsecretstore
+```
+
+4. Run your app
+
+You can check that `secretstores.json.jsonsecretstore` component is loaded and redis server connects successfully by looking at the log output when using the dapr `run` command with the `--enable-json-secretstore` flag.
+
+> The JSON secret store is in no way recommended for production environments and therefore is disabled by default. Using the `--enable-json-secretstore` flag sets the **DAPR_ENABLE_JSON_SECRET_STORE** environmnet variable to **1** so the store is enabled in your developemnt or demo box.
+
+Here is the log when you run [HelloWorld sample](https://github.com/dapr/samples/tree/master/1.hello-world) with JSON Secret secret store.
+
+```bash
+$ dapr run --enable-json-secretstore --app-id mynode --app-port 3000 --port 3500 node app.js
+
+ℹ️  Starting Dapr with id mynode on port 3500
+✅  You're up and running! Both Dapr and your app logs will appear here.
+
+...
+== DAPR == time="2019-09-25T17:57:37-07:00" level=info msg="loaded component jsonsecretstore (secretstores.json.jsonsecretstore)"
+== APP == Node App listening on port 3000!
+== DAPR == time="2019-09-25T17:57:38-07:00" level=info msg="loaded component statestore (state.redis)"
+== DAPR == time="2019-09-25T17:57:38-07:00" level=info msg="loaded component messagebus (pubsub.redis)"
+...
+== DAPR == 2019/09/25 17:57:38 redis: connecting to [redis]:6379
+== DAPR == 2019/09/25 17:57:38 redis: connected to [redis]:6379 (localAddr: x.x.x.x:62137, remAddr: x.x.x.x:6379)
+...
+```
+
+## References
+
+- [Secrets Component](../../concepts/secrets/README.md)
```

Comment:
Add another link to Secrets API https://github.com/dapr/docs/blob/master/reference/api/secrets_api.md
And another to the secrets sample: https://github.com/dapr/samples/blob/master/9.secretstore/README.md